package.json

```diff
@@ -4,6 +4,9 @@
     "repository": "git@github.com:ibexa/eslint-config-ibexa.git",
     "private": true,
     "dependencies": {
+        "@typescript-eslint/eslint-plugin": "^8.10.0",
```

Comment:
I'm not sure if we want to have `^` here, at least all other packages (added earlier) are hardcoded to specific version, probably to have better control over possible (even little) changes